resources/RemoveInactiveUsersRunner.php

```diff
@@ -57,36 +72,58 @@
     if ($lastLoginDate) { // null lastLoginDate check
         $interval = $today->diff($lastLoginDate);
     } else { // This might only be run once, since new users always have field filled.
-        echo "User has no lastLoginDate (it may have been a very long time.)\n";
-        echo "Deleting user.\n";
+        echo "Deleting this user as it has no last login date " .
+        "(it may have been a very long time). \n";
+        //writing to file to say who would have been deleted
+        file_put_contents("InactiveUsersToBeDeleted.txt", "The user " .
+        "with Id ".$user->getId()." would have been deleted as it " .
+        "has no last login date.\n", FILE_APPEND);
         deleteUser($user, $entityManager);
         echo "\n";
         // Move onto the next users.
         continue;
     }
 
-    $elapsedMonths = (int) $interval->format('%a') / 30;
-    echo 'Months elapsed since last login: ' . $elapsedMonths . "\n";
-
-    if ($elapsedMonths > $deletionThreshold) { // Delete user
-        echo "Deleting user\n";
-        deleteUser($user, $entityManager);
+    $elapsedMonths = (int) $interval->format('%a') / 30.4375; // 30.4375 is 365.25/12
+    echo 'Months elapsed since last login: ' . round($elapsedMonths,2) . "\n";
+    // Rounded months since last login to 2 decimal places for easier reading.
```

Comment:
this comment should preceed the code if refers to.

Reply:
ok.